matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/common/CommonTestHelper.kt

```diff
@@ -19,6 +19,8 @@ package org.matrix.android.sdk.common
 import android.content.Context
 import android.net.Uri
 import androidx.lifecycle.Observer
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement.runOnUiThread
```

Comment:
strange to use `internal` class here, but it appear that you fix the pb with UI thread like that...

Reply:
Took that from message from @onurays in android rooms. But we might need to dig more and see how to make test more simple it's painful to have to switch thread like that